app/views/posts/index.html.erb

```diff
@@ -1 +1,27 @@
-<h1>Posts of the user</h1>
\ No newline at end of file
+<%= render partial: "users/render", users: @users%>
+
+<% @posts.each do |post| %>
+<ul class="main-container">
+ <li class="blog">
+      <a href="<%= user_post_path(post.author_id, post.id) %>" class="blog">
+        <h2><%= post.title %></h2>
```

Comment:
- [ ] [Optional] Same here, It would be better professionally if you used the `link_to` method instead of the regular anchor tag `<a>`. see [this link](https://www.rubyguides.com/2019/05/rails-link_to-method/) for more info.